### PR TITLE
Revert "CRIU disable tests on ppc64le platform temporarily"

### PR DESCRIPTION
**Pull request by llxia**
Reverts eclipse-openj9/openj9#18469

related: https://github.com/eclipse-openj9/openj9/issues/18468

### CODE REVIEW

File: test/functional/cmdLineTests/criu/playlist.xml [hunk 1]
```diff
@@ -204,12 +192,6 @@
 			<variation>-XX:+DebugOnRestore -Xjit</variation>
 			<variation>-XX:+DebugOnRestore -Xjit:count=0</variation>
 		</variations>
-		<disables>
```
Comment (JasonFengJ9) [inline test/functional/cmdLineTests/criu/playlist.xml:207]: There is another `disable` section added later at L168, please remove it as well

**Comment by JasonFengJ9** [comment]
FYI @tajila

File: test/functional/cmdLineTests/criu/playlist.xml (hunk 1 above)
Comment (llxia) [reply test/functional/cmdLineTests/criu/playlist.xml:207]: This is a revert. I do not think I can directly change it. I will reenable L168 in a separate PR.

Comment (keithc-ca) [reply test/functional/cmdLineTests/criu/playlist.xml:207]: I expect you can (force-)push other changes to the branch behind, but if that other `<disable>` was added via a separate pull request, this should be clear that it reverts multiple changes.

Comment (llxia) [reply test/functional/cmdLineTests/criu/playlist.xml:207]: updated

**Review by JasonFengJ9 — APPROVED** [review]
LGTM

Thanks @llxia

**Comment by llxia** [comment]
@AdamBrousseau @keithc-ca could either of you review and merge the PR? Thanks

**Review by AdamBrousseau — APPROVED** [review]

**Pull request merged**